src/app/client/page.tsx

```diff
@@ -76,7 +76,11 @@ export default async function ClientPage({ searchParams }: { searchParams: { tok
         viewSettings={viewSettings}
       >
         <Suspense fallback={null}>
-          <AssigneeFetcher token={token} userType={UserType.CLIENT_USER} />
+          <AssigneeFetcher
+            token={token}
+            userType={UserType.CLIENT_USER}
+            isPreview={!!(tokenPayload.internalUserId && tokenPayload.clientId)}
```

Comment:
This will not work.

In company preview mode we won't have clientId, but a companyId that isn't "default" instead. This logic will not treat company preview as preview mode